src/routes/articlesRoute.js

```diff
@@ -22,5 +22,8 @@ articles.get('/articles/:slug/share/facebook', verifyToken, slugExist, shareArti
 articles.get('/articles/:slug/share/linkedin', verifyToken, slugExist, shareArticle, articlesController.share);
 articles.get('/articles/:slug/share/email', verifyToken, slugExist, shareArticle, articlesController.share);
 
+articles.get('/articles', verifyToken, articlesController.getAllArticles);
```

Comment:
In our previous code review session we said the only functions that we shall be accepting in your routes are: url, auth-middleware and controller. Kindly lets follow what we agreed.